INSTALL.md

```diff
@@ -96,13 +96,21 @@ In order to install the llvm (clang) toolchain, you may have to follow instructi
 ### OS X
 When building on OSX, here's some dependencies you'll need:
 - Install xcode
-- Install homebrew
-- brew install libsodium
-- brew install libtool
-- brew install automake
-- brew install pkg-config
-- brew install libpqxx *(If ./configure later complains about libpq missing, try PKG_CONFIG_PATH='/usr/local/lib/pkgconfig')*
-- brew install parallel (required for running tests)
+- Install [homebrew](https://brew.sh)
+- `brew install libsodium`
+- `brew install libtool`
+- `brew install autoconf`
+- `brew install automake`
+- `brew install pkg-config`
```

Comment:
I didn't need to install `libsodium`, `libtool`, or `pkg-config`, when I last setup on a fresh mac. They might not be required anymore, or maybe they are dependencies of other things here and are getting pulled in automatically. I haven't removed them from the list because I don't know for certain they are required or not.